students/JaNiece/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="index.css">
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title> My Webpage </title>
+</head>
+
+<body>
+  <header>
+    <nav>
+      <img src="images/sm.jpeg" class="logo">
+      <a href="index.html"> Home </a>
+      <a href="life.html"> Learn More </a>
+      <a href="goals.html"> My Goals </a>
+      <a href="contact.html" class="right"> Contact </a>
+    </nav>
+  </header>
+
+  <section class="about">
+    <h2> About </h2>
+    <p>
+      My Name is JaNiece Levelston. I was born and raised in Muskegon, MI on April 13. I am 24, I have 1 sister, 3 brothers and a son (dog) name Ace! People would describe me as generous, very quiet, smart, and goofy. I tend to stay to myself though. But anyways, on to the good stuff! I am a huge marvel fan, Spider-Man is my favorite as you can see. i have a huge collection in my room and can't wait to add more. My favorite food is Fettuccine Alfredo. My favorite color is red. My favorite season is Fall, like seriously haha my closet is filled with fall clothes. I'm not an outdoors person, so I prefer fun indoor activities such as bowling, indoor go-carting, etc. Other then indoor activities, I love watching movies, playing video games, dancing, coding, and playing basketball. There's more, but those are the ones I do the most. My dream job is to one day become a web developer and taking this class is a step closer to that dream.
+    </p>
+  </section>
+
+<footer>
+  <a href="../index.html">Back to main page</a>
```

Comment:
❓ When I try and click this link in the browser, it doesn't take me back to the main index.html page... any ideas why this might be happening?